src/components/body-helper.js

```diff
@@ -34,25 +35,24 @@ AFRAME.registerComponent("body-helper", {
   init: function() {
     this.system = this.el.sceneEl.systems["hubs-systems"].physicsSystem;
     this.alive = true;
+    this.uuid = -1;
     this.system.registerBodyHelper(this);
   },
 
   init2: function() {
-    this.body = new Body(this.data, this.el.object3D, this.system.world);
-    this.system.addBody(this.body);
+    this.el.object3D.updateMatrices();
+    this.uuid = this.system.addBody(this.el.object3D, this.data);
```

Comment:
This `uuid` is used as a key in several objects holding info associated with this body -- any reason not to use this (already unique) `body-helper` as the key in `Map` objects (which, unlike objects, allow keys of any type)? It seems like everywhere that uses the `uuid` gets it by first getting a reference to the `body-helper`